docs/content/_navigation.json

```
@@ -224,17 +228,22 @@
           },
           {
             "title": "Jobs",
+            "path": "/concepts/ops-jobs-graphs/jobs",
             "children": [
               {
-                "title": "Jobs",
-                "path": "/concepts/ops-jobs-graphs/jobs"
+                "title": "Asset jobs",
+                "path": "/concepts/assets/asset-jobs"
+              },
+              {
+                "title": "Op jobs",
+                "path": "/concepts/assets/op-jobs"
```

```suggestion
                "path": "/concepts/ops-jobs-graphs/op-jobs"
```
wrong link caused 404

---

Fixed!